### PR TITLE
Update Netlify CMS

### PR DESCRIPTION
**Pull request by covingtondoan**
Based heavily on current netlify-templates/jekyll-base repo.

### CODE REVIEW

File: admin/index.html [hunk 1]
```diff
@@ -1,24 +1,14 @@
...
-
   <title>Content Manager</title>
-  <!-- Include the stylesheets from your site here -->
-  <link rel="stylesheet" href="/css/main.css">
-  <link rel="stylesheet" href="//cms.netlify.com/assets/cms.css" />
-  <!-- Include a CMS specific stylesheet here -->
 
-  <!-- Set the CMS environment -->
-  <script>window.CMS_ENV="{{site.env}}";</script>
+  <link rel="stylesheet" href="https://unpkg.com/netlify-cms@^0.3/dist/cms.css" />
```
Comment (erquhart) [inline admin/index.html]: Can you bump this and the script to `^0.4`?

**Review by erquhart — CHANGES_REQUESTED** [review]
Thanks for this!! Just need the cms version bumped, otherwise awesome improvements.

**Review by erquhart — APPROVED** [review]
LGTM

**Comment by verythorough** [comment]
🎉 LGTM

**Pull request merged**